tools/templates/L10N/IT

```diff
@@ -0,0 +1,411 @@
+#infix-ge          ge
+#infix-gt          gt
+#infix-le          le
+#infix-lcm         lcm
+#infix-leg         leg
+#infix-lt          lt
+#infix-max         max
+#infix-min         min
+#infix-minmax      minmax
+#infix-mod         mod
+#infix-ne          ne
+#infix-notandthen  notandthen
+#infix-o           o
+#infix-or          or
+#infix-orelse      orelse
+#infix-unicmp      unicmp
+#infix-x           x
+#infix-X           X
+#infix-xx          xx
+#infix-Z           Z
+#infix-ff^         ff^
+#infix-fff^        fff^
+#infix-(cont)      (cont)
+#infix-(elem)      (elem)
+#infix-^ff         ^ff
+#infix-^fff        ^fff
+#infix-^ff^        ^ff^
+#infix-^fff^       ^fff^
+
+# KEY             TRANSLATION
+#meta-R  R
+#meta-X  X
+#meta-Z  Z
+
+# KEY             TRANSLATION
+#modifier-for      for
+#modifier-given    given
+#modifier-if       if
+#modifier-unless   unless
+#modifier-until    until
+#modifier-when     when
+#modifier-while    while
+#modifier-with     with
+#modifier-without  without
+
+# KEY        TRANSLATION
+#multi-multi  multi
+#multi-only   only
+#multi-proto  proto
+
+# KEY            TRANSLATION
+#package-class    class
+#package-grammar  grammar
+#package-knowhow  knowhow
+#package-module   module
+#package-native   native
+#package-package  package
+#package-role     role
+
+# KEY           TRANSLATION
+#phaser-BEGIN    BEGIN
+#phaser-CATCH    CATCH
+#phaser-CHECK    CHECK
+#phaser-CLOSE    CLOSE
+#phaser-CONTROL  CONTROL
+#phaser-DOC      DOC
+#phaser-END      END
+#phaser-ENTER    ENTER
+#phaser-FIRST    FIRST
+#phaser-INIT     INIT
+#phaser-KEEP     KEEP
+#phaser-LAST     LAST
+#phaser-LEAVE    LEAVE
+#phaser-NEXT     NEXT
+#phaser-PRE      PRE
+#phaser-POST     POST
+#phaser-QUIT     QUIT
+#phaser-UNDO     UNDO
+
+# KEY       TRANSLATION
+#prefix-not  not
+#prefix-so   so
+
+# KEY              TRANSLATION
+#routine-method     method
+#routine-sub        sub
+#routine-regex      regex
+#routine-rule       rule
+#routine-submethod  submethod
+#routine-token      token
+
+# KEY            TRANSLATION
+#scope-anon       anon
+#scope-augment    augment
+#scope-constant   constant
+#scope-has        has
+#scope-HAS        HAS
+#scope-my         my
+#scope-our        our
+#scope-state      state
+#scope-supersede  supersede
+#scope-unit       unit
+
+# KEY                 TRANSLATION
+#stmt-prefix-also      also
+#stmt-prefix-do        do
+#stmt-prefix-eager     eager
+#stmt-prefix-gather    gather
+#stmt-prefix-hyper     hyper
+#stmt-prefix-lazy      lazy
+#stmt-prefix-quietly   quietly
+#stmt-prefix-race      race
+#stmt-prefix-react     react
+#stmt-prefix-sink      sink
+#stmt-prefix-start     start
+#stmt-prefix-supply    supply
+#stmt-prefix-try       try
+
+# KEY      TRANSLATION
+#term-self  self
+#term-nano  nano
+term-now   gia
+#term-rand  rand
+term-time  tempo
+
+# KEY          TRANSLATION
+trait-does     fa
+trait-handles  gestisce
+trait-hides    nasconde
+trait-is       e
+trait-of       da
```

Comment:
`of` can be translated in Italian as `da` (origin, procedence) or `di` (possessive). I'm going for origin here.

Reply:
Yeah, this is a general issue.  That's why I made the "categories" because the same word in english, could have different translations in other languages.

Such as `if` as a block initiator, and `if` as a modifier.